### DISCUSSION TITLE
Delete some models or ignore while generating?

### DISCUSSION THREAD

**Discussion by adsee42** [MichalLytek/typegraphql-prisma#125] [Q&A] [answered]
I'm connecting prisma to an existing postgresql db, which I can't modify the tables (only crud on existing tables).
The db contains 50+ tables and with `prisma introspect`, both prisma schema and graphql schema are generated for all tables. 

Almost 2/3 of the tables are not needed, so is it safe to delete some of the generated graphql types and queries and mutations?

Or is there any options to ignore some table while generating the schemas?

**Comment by MichalLytek** [comment, marked as answer]
You can delete the not used code until it not breaks while compiling 😅 
Be aware of relations resolvers.

Omit support for models will be added in some time, for now it only supports fields ommiting:
https://github.com/MichalLytek/typegraphql-prisma#hiding-prisma-model-field-in-graphql-schema
